CustomHats.json

```diff
@@ -173,6 +173,29 @@
             "reshashc": "93275fadbf159ae3319cb42b1e458df2",
             "resource": "brain_adaptive.png"
         },
+        {
+            "author": "Niggo",
+            "backresource": "Little_Devil_back.png",
+            "climbresource": "Little_Devil_climb.png",
+            "condition": "None",
+            "name": "Little Devil",
+            "package": "Community Hats",
+            "reshasha": "dd982e72b0d8eea9813b3de5608b98a9",
+            "reshashb": "dc5db6d02434bd8dfcae981a6f399310",
+            "reshashc": "96b81d4d937d9e78f12a26217ee5f67c",
+            "resource": "Little_Devil.png"
+        },
+        {
+            "adaptive": true,
+            "author": "Hoax",
+            "backresource": "Hoax_arms_adaptive_back.png",
+            "condition": "None",
+            "name": "Arms",
+            "package": "Community Hats",
+            "reshasha": "f073cd06862d50c057952868a9e4b366",
+            "reshashb": "af91adea15c6bb98c31b46d295327d8b",
+            "resource": "Hoax_arms_adaptive.png"
+        },

```

Comment:
- New additions need to be added to the end of the array and not inbetween to keep the hat IDs in order.
- `_` represents the seperator for options. If you want to substitute spaces use `-` instead.
- It would also be better to have PRs for individual hats, rather than bundling them. (I personally dislike if *hats* implement arms, as it looks very awkward in animations etc.)

Reply:
thanks for your feedback

I'll change name from `Little_Devil_climb.png` to `Little-Devil_climb.png` ? 
Or maybe camelcase `littleDevil_climb.png `? 

"I personally dislike if hats implement arms"
If you want to have choice to have arms or not it's the only way I think